projects/canopy/src/lib/carousel/carousel.component.ts

```diff
@@ -58,8 +69,31 @@ export class LgCarouselComponent implements AfterContentInit {
     }
   }
 
+  setAutoPlayInterval(): void {
+    this.pausableTimer$ = defer(() => {
+      return interval(this.autoPlayDelay).pipe(
+        takeUntil(this.unsubscribe),
+        withLatestFrom(this.pause),
+        filter(([v, paused]) => !paused),
```

Comment:
Not sure what `v` is in this `filter` operator? Is the interval? Just wondering if it should be `interval` instead?

Reply:
Its not needed. Removed